test/ci-operator-configresolver-integration/main.go

```diff
@@ -0,0 +1,88 @@
+package main
```

Comment:
There is no need to create a new binary to run and compare the results. You can be advised from the `pj-rehearse` and `ci-operator` integration tests. Since you are hardcoding the configs, you can have a JSON expected file and `diff` it, while you will be able to `GET` via curl in the bash script.

Reply:
I was intending on using this to more easily modify files and verify that the resolver picks up the changes automatically in future tests (this functionality will be added in a future PR). Thinking about it though, we could do this in bash a bit more easily, we would just need more input files we can copy over/modify and expected result files to compare against. I'll change it to simple bash diffs for now.